tests/QueryType/ManagedResources/RequestBuilder/StopwordsTest.php

```diff
@@ -99,4 +99,30 @@ public function testExists()
         $this->assertEquals($term, $command->getTerm());
         $this->assertEquals('', $command->getRawData());
     }
+
+    public function testReservedCharacters()
```

Comment:
These are just unit tests that don't prove that the double encoding really works with Solr. We already have Integration tests that query a real Solr server on travis. I think such a test would be preferable here.

Reply:
A real Solr test would be preferable. I just felt some test was necessary. The double encoding (if we settle on it) is so non-obvious that it's a prime candidate for regressions.

Reply:
@thomascorthals Can you solve the conflicts with master and an integration test?